c/src/ml-api-internal.h

```diff
@@ -73,12 +73,12 @@ typedef enum
  * @brief Enumeration for machine_learning feature.
  */
 typedef enum {
-  ML_FEATURE  = 0,
+  ML_FEATURE = 0,
   ML_FEATURE_INFERENCE,
   ML_FEATURE_TRAINING,
   ML_FEATURE_SERVICE,
 
-  ML_FEATURE_UNKNOWN
+  ML_FEATURE_MAX
```

Comment:
If you have additional FEATURE enum, MAX will be updated. Be sure that ML_FEATURE_MAX won't be exposed to public API and public apps won't have this value embedded in their binaries.